services/wallet/service.go

```diff
@@ -518,54 +522,54 @@ func (service *Service) LinkGeminiWallet(ctx context.Context, walletID uuid.UUID
 		ctx = context.WithValue(ctx, appctx.CustodianRegionsCTXKey, &cr)
 	}
 
-	// perform an Account Validation call to gemini to get the accountID
+	// Validate the account with Gemini. If a wallet has previously been linked but the country is now invalid
+	// i.e. validate account returns errorutils.ErrInvalidCountry then we can ignore this
+	// error and allow the account to link due to its prior successful linking.
+	// If there is no prior linking then we should return the original error.
```

Comment:
All good with this comment, but I am not sure it explains why this behaviour is here. Or am I reading it poorly? Shall we put a link to the issue or add a sentence to tell why we are doing this?

Reply:
This is suppose to explain it but maybe - _"If a wallet has previously been linked but the country is now invalid i.e. validate account returns errorutils.ErrInvalidCountry then we can ignore this error and allow the account to link due to its prior successful linking."_ maybe its not clear enough

Reply:
Thank you, I believe now it's clearer. 👍